### PR TITLE
Implemented Forgot Password and Signup 

### PR DESCRIPTION
**Pull request by SRV30**
Implemented Forgot Password and Signup **issue no #318** 
1. Forgot Password 
image
 
image 

image 

image

2.  Signup
Capture - Old 

image - New

3. Remove some problems
Capture1

### CODE REVIEW

**Comment by SRV30** [comment]
is this ok?

**Comment by SRV30** [comment]
@maniyaom, please review it

**Comment by Khizarshah01** [comment]
@SRV30 I saw your pull request! However, another pull request was accepted recently, and it looks like your changes are now behind the latest updates. Could you please update your branch to reflect these changes?

**Comment by SRV30** [comment]
> @SRV30 I saw your pull request! However, another pull request was accepted recently, and it looks like your changes are now behind the latest updates. Could you please update your branch to reflect these changes?

ok

**Comment by SRV30** [comment]
@Khizarshah01 now check it

File: src/index.js
Comment (Khizarshah01) [inline src/index.js:1]: done!

File: src/Pages/ForgotPassword.js
Comment (Khizarshah01) [inline src/Pages/ForgotPassword.js:1]: done!

**Review by Khizarshah01 — COMMENTED** [review]
The code looks good overall! It meets the project requirements and functions as intended. Here are a couple of suggestions:

    Variable Names: Consider using more descriptive names in some areas for clarity.
    Testing: Ensure there are adequate test cases, especially for edge cases.

Great job!

**Comment by SRV30** [comment]
@Khizarshah01 will u merge it?

**Comment by maniyaom** [comment]
@SRV30 everything is perfect just use the primary color of the website.
Change button's color to #f84464

image

**Comment by SRV30** [comment]
> @SRV30 everything is perfect just use the primary color of the website. Change button's color to #f84464
> 
> image

Done. Change the color to #f84464 for all the changes I made. Check it. Thank You.

**Comment by Khizarshah01** [comment]
@maniyaom

**Pull request merged**